### PR TITLE
Add an entry to eclipses for scala-ide

### PR DESCRIPTION
**Pull request by earldouglas**
I think `eclipses` is the appropriate place for this, since it follows the pattern of the others.  Scala IDE is the regular Eclipse, bundled with the Scala IDE [additions](http://scala-ide.org/download/current.html).

### CODE REVIEW

File: pkgs/applications/editors/eclipse/default.nix [hunk 1]
```diff
@@ -271,4 +271,19 @@ in {
     };
   };
 
+  scala_ide = buildEclipse {
+    name = "scala-ide";
+    description = "Scala IDE for Eclipse";
+    sources = {
+      "x86_64-linux" = fetchurl {
+          url = http://downloads.typesafe.com/scalaide-pack/4.0.0.vfinal-luna-211-20150305/scala-SDK-4.0.0-vfinal-2.11-linux.gtk.x86_64.tar.gz;
+          sha256 = "b65c5e8160e72c8389537e9e427138e6daa2065f9df3a943a86e40dd1543dd83";
```
Comment (earldouglas) [inline pkgs/applications/editors/eclipse/default.nix]: Why are these hashes longer than the other ones in this file?  I created them with `sha256sum` -- is there another way?

Comment (joachifm) [inline pkgs/applications/editors/eclipse/default.nix]: @earldouglas The difference in length is due to a different representation of the underlying binary digest, it shouldn't matter.

**Comment by rycee** [comment]
I think this PR could be closed. There is a derivation named `eclipse-scala_sdk-4.0.0` now. It is a bit out of date, though. If you want a more recent Scala-IDE I would suggest to use `eclipse-platform` package and add the `scala` plugin as described in the [nixpkgs manual](https://nixos.org/nixpkgs/manual/#sec-eclipse). I believe this has pretty much everything from Scala-IDE except the branding.

**Comment by earldouglas** [comment]
@rycee Thanks!

**Pull request closed without merging**